src/common/components/home/FeaturedBuyNowListingsSection.tsx

```diff
@@ -0,0 +1,106 @@
+import { VFC } from 'react';
+import { NFTCard } from 'pages/profiles/[publicKey]/nfts';
+import { gql, useQuery } from '@apollo/client';
+import { HomeSection, HomeSectionCarousel } from 'pages/home-v2-wip';
+
+// TODO replace hardcoded list with gql query result
+const featuredListings: {address: string, marketplace: string}[] = [
+  {address: '9CZmL7zc87Qc4d8svJdVHjmmd5V9TVhGAqvziv7ibV1K', marketplace: 'junglecats'},
```

Comment:
It's just the jungle cats marketplace for now?

Reply:
yeah this is just a placeholder. I had started on work on this and then put it aside to work on another section, but didnt want to lose the (interdependent) work.

Reply:
gotcha